dockers/docker-fpm/Dockerfile.j2

```diff
@@ -2,7 +2,7 @@ FROM docker-base
 
 RUN apt-get update
 
-RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4
+RUN apt-get install -y libdbus-1-3 libdaemon0 libjansson4 python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml
```

Comment:
In this version let's keep python dependencies out of base image. Let's discuss later and make changes if needed in a seperated PR.